api.raml

```diff
@@ -38,40 +38,30 @@ securitySchemes:
         description: Number of rows to skip from the result
     responses:
       200:
```

Comment:
All these `displayName` removals were to make the raml actually spec compliant.  This was done because there were generation issues and I was attempting to remove all possible causes.